libafl_qemu/src/qemu/qemu_config.rs

```diff
@@ -301,20 +308,23 @@ impl<R: AsRef<Path>> From<R> for Program {
     }
 }
 
-#[derive(Debug, Clone, libafl_derive::Display, TypedBuilder)]
+#[derive(Debug, Clone, libafl_derive::Display, TypedBuilder, Getters)]
```

Comment:
I mean I _feel_ like we can just make the fields `pub(create)` and get around the extra dependency here(?)

Reply:
(even though actual getters seem a bit more softare-engineering-y)

Reply:
The problem of making them `pub` is that they could also be set and the struct even created bypassing everything...
about restricting to `(crate)` I don't have a strong opinion atm but might be handy to make the config available outside (?)

Reply:
Making it `pub` would _probably_ be fine, too - it's not like the library is for beginners :D. 
But ok sounds reasonable